input/haleu/analysis/df_to_csv.py

```diff
@@ -1,6 +1,9 @@
 import pandas as pd
-
+import sys
 import cymetric as cm
+import cProfile
+
+sys.path.insert(0, '../../../scripts')
```

Comment:
Why do we need this on the system path? Are you calling a script from within python?